tests/conftest.py

```diff
@@ -180,7 +180,7 @@ def valid_simple_ml_predictor_data(valid_gem_data_source_dict):
     z = RealDescriptor("z", 0, 100, "")
     return dict(
         module_type='PREDICTOR',
-        status='VALID',
+        status='READY',
```

Comment:
Why are we changing these?  I thought they do come back from the backend with `status='VALID'`.

Reply:
Nope, it's 'READY'. You can check the json directly, but the easiest way to see it is to go the Predictors list in the UI